### PR TITLE
Implement scipy.stats.logistic

### LINKED ISSUE #1986: Could we have support for scipy.stats.logistic?
In #324 we got support for `logit` and `expit` from `scipy.special`. This provides all the building blocks to also support `scipy.stats.logistic` (in particular, it’s members `cdf`, `pdf`, `ppf`, `logcdf` and `logpdf`). Having this would enable doing maximum-likelihood estimation of logistic regression using `scipy.optimize`.

### PR DESCRIPTION
**Pull request by TuanNguyen27**
Resolves #1986

### CODE REVIEW

File: jax/scipy/stats/logistic.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Copyright 2018 Google LLC
```
Comment (mattjj) [inline jax/scipy/stats/logistic.py]: Change to 2020?

**Review by mattjj — APPROVED** [review]
This is beautifully done. Thanks for the contribution!

File: jax/scipy/stats/logistic.py (hunk 1 above)
Comment (TuanNguyen27) [reply jax/scipy/stats/logistic.py]: done !

**Comment by TuanNguyen27** [comment]
@mattjj pinging you again in case this is ready to merge !

**Pull request merged**